lib/devicetrust/authz/authz.go

```diff
@@ -59,8 +59,10 @@ func IsSSHDeviceVerified(cert *ssh.Certificate) bool {
 		cert.Extensions[teleport.CertExtensionDeviceCredentialID] != ""
 }
 
-// HasDeviceTrustExtensions returns true if the profile extensions contain
-// all required device extensions.
+// HasDeviceTrustExtensions returns true if the certificate's extension names
+// include all the required device-related extensions.
+// Unlike IsSSHDeviceVerified, this function operates on a list of extensions,
+// such as those in [ProfileStatus.Extensions].
```

Comment:
I don't think this link works, ProfileStatus lacks a package name (and we don't import that package here).

```suggestion
// such as those in lib/client.ProfileStatus.Extensions.
```

Reply:
Thanks, I didn't realize that only imported packages can be referenced in godoc.

Reply:
https://go.dev/doc/comment#doclinks